ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/PendingTransactions.java

```diff
@@ -73,6 +74,7 @@
       new TreeSet<>(
           comparing(TransactionInfo::isReceivedFromLocalSource)
               .thenComparing(TransactionInfo::getSequence)
+              .thenComparing(TransactionInfo::getGasPrice)
```

Comment:
We want to order by gas prices first and then by the order in which they came in.

Reply:
ah, I see, I actually am just now figuring out how to run the tests for besu / reading into the first contribution workflow =)

I should have something shortly!

Reply:
I think I got the new ordering right =) I managed to run the test locally and they worked! 
I am not sure why they don't build via git-actions though?